### DISCUSSION TITLE
How to mark as Read Only

### DISCUSSION THREAD

**Discussion by textual** [visgl/react-google-maps#507] [Q&A] [answered]
I'd like to show a mini map of a location on a map and I don't want the user to move the map, zoom or anything. Is there a way to mark the map as read only?

**Comment by usefulthink** [comment, marked as answer]
you can set `center` and `zoom` to define the view and then add the `controlled` prop. That should disable all user-inputs. So

```tsx
<Map center={...} zoom={...} controlled></Map>
```

But also checkout the Static Maps API, which is quite a bit cheaper and faster: https://developers.google.com/maps/documentation/maps-static/overview

**Reply by textual** [reply]
Ah, i was setting default center and it was allowing it to move. the Static Maps API looks promising as well. Thanks!
